plugins/CorePluginsAdmin/Commands/InstallPlugin.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\CorePluginsAdmin\Commands;
+
+use Piwik\Container\StaticContainer;
+use Piwik\Plugin\ConsoleCommand;
+use Piwik\Plugin\Manager;
+use Piwik\Plugins\CorePluginsAdmin\PluginInstaller;
+use Piwik\Plugins\Marketplace\Marketplace;
+
+/**
+ * plugin:install console command.
+ */
+class InstallPlugin extends ConsoleCommand
+{
+    protected function configure(): void
+    {
+        $this->setName('plugin:install');
+        $this->setDescription('Install a plugin.');
+        $this->addOptionalArgument('plugin', 'The plugin name you want to install. Multiple plugin names can be specified separated by a space.', null, true);
+    }
+
+    protected function doExecute(): int
+    {
+        $input = $this->getInput();
+        $output = $this->getOutput();
+        $pluginManager = Manager::getInstance();
+
+        $pluginNames = $input->getArgument('plugin');
+
+        foreach ($pluginNames as $pluginName) {
+            if ($pluginManager->isPluginInstalled($pluginName)) {
+                $output->writeln(sprintf('<comment>The plugin %s is already installed.</comment>', $pluginName));
+                continue;
+            }
+
+            if (!$pluginManager->isPluginInFilesystem($pluginName) && !Marketplace::isMarketplaceEnabled()) {
```

Comment:
What is the `isPluginInFilesystem` check here meant for?
If Marketplace is disabled I guess the commend shouldn't be doing anything at all, right? So guess the `isMarketplaceEnabled` check could be done before iterating over `$pluginName`